addons/website_sale/i18n/fr.po

```diff
@@ -1717,6 +1717,11 @@ msgstr "Permettez l'achat directement depuis le catalogue avec :"
 msgid "List displaying"
 msgstr "Affichage en liste"
 
+#. module: website_sale
+#: model:ir.ui.view,arch_db:website_sale.address
```

Comment:
don't put translations in your code, you can modify the .pot file but the .po files are managed on transifex